src/omotes_simulator_core/adapter/transforms/mappers.py

```diff
@@ -167,33 +149,17 @@ def to_entity(self, network: Network) -> Tuple[List[AssetAbstract], List[Junctio
                         index = [py_asset_temp.asset_id for py_asset_temp in py_assets_list].index(
```

Comment:
Kan je even wat comments plaatsen wat hier ook alweer gebeurt (ook al is het niet jouw stuk). Het is soms erg lastig om weer te snappen waarom bepaalde statements zijn toegevoegd.